midp/midp.js

```diff
@@ -927,7 +927,17 @@ var MIDP = (function() {
       var nativeEventQueue = nativeEventQueues[isolateId];
 
       if (nativeEventQueue.length !== 0) {
-        copyEvent(nativeEventQueue.shift(), nativeEvent);
+        var ev = nativeEventQueue.shift();
+        if (ev.type === PEN_EVENT
+         && ev.intParam1 === DRAGGED) {
+          while (nativeEventQueue.length !== 0
+              && nativeEventQueue[0].type === PEN_EVENT
+              && nativeEventQueue[0].intParam1 === DRAGGED) {
+            ev = ev2;
```

Comment:
> There's probably something missing here, ev2 isn't defined.

Not sure what happened here. I updated the patch with a new approach so let's discuss there.